test/specs/js-file.js

```diff
@@ -162,6 +162,61 @@ describe('modules/js-file', function() {
             assert(!file.isEnabledRule('validateQuoteMarks', 4));
             assert(file.isEnabledRule('validateQuoteMarks', 7));
         });
+
+        describe('single line trailing comment', function() {
+            it('should ignore a single line', function() {
+                var file = createJsFile([
+                    'var x = "1";',
+                    'var y = "1"; // jscs: ignore validateQuoteMarks',
+                    'var z = "1";',
+                ].join('\n'));
+                assert(file.isEnabledRule('validateQuoteMarks', 1));
+                assert(!file.isEnabledRule('validateQuoteMarks', 2));
+                assert(file.isEnabledRule('validateQuoteMarks', 3));
+            });
+
+            it('should not re-enable rules', function() {
+                var file = createJsFile([
+                    'var a = "1";',
+                    '// jscs: disable validateQuoteMarks',
+                    'var b = "1"; // jscs: ignore validateQuoteMarks',
+                    'var c = "1";',
+                ].join('\n'));
+                assert(file.isEnabledRule('validateQuoteMarks', 1));
+                assert(!file.isEnabledRule('validateQuoteMarks', 2));
+                assert(!file.isEnabledRule('validateQuoteMarks', 3));
+                assert(!file.isEnabledRule('validateQuoteMarks', 4));
+            });
+
+            it('should also work with multiple rules', function() {
+                var file = createJsFile([
+                    '// jscs: disable validateQuoteMarks',
+                    'var a = "1"; // jscs: ignore validateQuoteMarks, anotherRule',
```

Comment:
I think @markelog means `//jscs: ` in that it doesn't have a space after the `//` like in `// jscs: ignore validateQuoteMarks`